localstack/aws/connect.py

```diff
@@ -27,10 +29,36 @@
 from localstack.utils.aws.aws_stack import get_local_service_url, get_s3_hostname
 from localstack.utils.aws.client_types import ServicePrincipal, TypedServiceClientFactory
 from localstack.utils.aws.request_context import get_region_from_request_context
+from localstack.utils.patch import patch
 from localstack.utils.strings import short_uid
 
 LOG = logging.getLogger(__name__)
 
+DEFAULT_DELAY = 5 if os.environ.get("TEST_TARGET") == "AWS_CLOUD" else 1
+DEFAULT_MAX_ATTEMPTS = 60 if os.environ.get("TEST_TARGET") == "AWS_CLOUD" else 120
```

Comment:
Might be better actually to use some monkeypatching here in the test setup instead. wdyt?

Reply:
Update: I've made it configurable now and only switch to aws defaults for tests in `tests/aws/`